manifests/nodejs.yml

```diff
@@ -390,7 +390,7 @@ tests/:
         '*': *ref_3_19_0
         nextjs: missing_feature
       Test_Suspicious_Request_Blocking:
-        '*': missing_feature (v3.19.0, but test is not implemented)
```

Comment:
`v3.19.0` is not the correct version since this test relies on path param blocking